packages/eth2.0-utils/src/bytes.ts

```diff
@@ -17,6 +17,15 @@ export function intToBytes(value: BN | number, length: number): bytes {
   }
 }
 
+export function bytesToInt(value: bytes): number {
+  const length = value.length;
+  let result = 0;
+  for (let i = 0; i < length; i++) {
+    result += value[i] * 2 ** (8 * i);
+  }
+  return result;
+}
```

Comment:
is this le or be? might wanna put that in comment or even in method name

Reply:
should be LE, matching `intToBytes` above

Reply:
I've just added comment to method.